### PR TITLE
feature/typescript

### PR DESCRIPTION
**Pull request by Musa-Msomi**
Integrated Typescript to my flight tracker web app. Please review and let me know of any changes that may need to be made

### CODE REVIEW

**Comment by Musa-Msomi** [comment]
Howzit guys @RBocibo @KBore2 can you please give this PR a look as soon as you have a gap

File: webpack.config.js [hunk 1]
```diff
@@ -38,6 +41,12 @@ module.exports = {
                 test: /\.(png|svg|jpg|jpeg|gif)$/i,
                 type: 'asset/resource'
             },
+            {
+
```
Comment (RBocibo) [inline webpack.config.js]: remove extra spacing, and format you file

**Review by RBocibo — APPROVED** [review]
Approved with comments, fix them then merge

File: webpack.config.js (hunk 1 above)
Comment (Musa-Msomi) [reply webpack.config.js]: Done. Thank you

**Pull request merged**